r/rsparkling/R/package.R

```diff
@@ -1,13 +1,24 @@
-
 #' @import sparklyr h2o utils
 NULL
 
 # define required spark packages
 spark_dependencies <- function(spark_version, scala_version, ...) {
+  sw_version = getOption("rsparkling.sparklingwater.version", default = "1.6.7")
```

Comment:
I'm thinking if it wouldn't be easier to throw error - warn the user - that the version has to be set as `+options(rsparkling.sparklingwater.version = "2.0.0")` in order to use sparkling-water. It's more explicit and we don't have to update default version.

Because like this, we would need to update default version with each sparkling-water release ( right now it should be 2.0.0 )

What do you think @navdeep-G, is this doable ? What do you @mmalohlava think ? I know we discussed that in the office, so wanted to know your input on this. 

btw: thanks @navdeep-G for this!